doc/adr/Which-branch-holds-what-is-reflected-in-live.md

```diff
@@ -0,0 +1,112 @@
+# ADR - Which branch holds what is reflected in Live?
```

Comment:
Which ADR is this going to supposedly be? And the file name itself will need to be numbered too.

Reply:
Although I assume you will do this last 🤔.